singularity-opac/neutrinos/spiner_opac_neutrinos.hpp

```diff
@@ -130,17 +132,19 @@ class SpinerOpacity {
 
   // DataBox constructor. Note that this constructor *shallow* copies
   // the databoxes, so they must be managed externally.
-  SpinerOpacity(const Spiner::DataBox &lalphanu, const Spiner::DataBox ljnu,
-                const Spiner::DataBox lJ, const Spiner::DataBox lJYe)
+  SpinerOpacity(const Spiner::DataBox &lalphanu, const Spiner::DataBox &lAlphanu,
+                const Spiner::DataBox ljnu, const Spiner::DataBox lJ,
+                const Spiner::DataBox lJYe)
       : memoryStatus_(impl::DataStatus::OnHost), lalphanu_(lalphanu),
-        ljnu_(ljnu), lJ_(lJ), lJYe_(lJYe) {}
+        lAlphanu_(lAlphanu), ljnu_(ljnu), lJ_(lJ), lJYe_(lJYe) {}
```

Comment:
I'm having second thoughts about actually introducing a separate table, rather than just changing the API but keep only 4 tables. The Burrows tables don't look any different between Alpha and alpha. Maybe we should save space and only use one table. What do you think @brryan ?

Reply:
You're right, I shouldn't add data just for prettiness reasons. Doubt we'll ever care about the distinction here. What do you think of this now?

Also wow I forgot about this PR for a long time.

Reply:
Cool thanks @brryan and no problem. :)